examples/hybrid/bubble_3d.jl

```diff
@@ -317,6 +312,12 @@ sol = @timev solve(
     progress_message = (dt, u, p, t) -> t,
 );
 
+if haskey(ENV, "CI_PERF_SKIP_RUN") # for performance analysis
+    throw(:exit_profile)
```

Comment:
should this just be `exit()`?

Reply:
oh, you catch it in the other script: in that case, maybe put the rest in a block?

Reply:
I'm not sure I follow. FWIW, these lines were only moved